prov/tcp/src/xnet_attr.c

```diff
@@ -50,9 +50,10 @@
 
 #define XNET_TX_OP_FLAGS \
 	(FI_INJECT | FI_INJECT_COMPLETE | FI_TRANSMIT_COMPLETE | \
-	 FI_DELIVERY_COMPLETE | FI_COMMIT_COMPLETE | FI_COMPLETION)
+	 FI_DELIVERY_COMPLETE | FI_COMMIT_COMPLETE | FI_COMPLETION | \
+	 TCP_NO_CONNECT)
 
-#define XNET_RX_OP_FLAGS (FI_COMPLETION)
+#define XNET_RX_OP_FLAGS (FI_COMPLETION | TCP_NO_CONNECT)
```

Comment:
It doesn't look like there's any connection look up or handling of this flag on the rx side. I think this is probably just a tx flag